sphinx_rtd_theme/layout.html

```diff
@@ -7,10 +7,11 @@
   {%- set titlesuffix = "" %}
 {%- endif %}
 {%- set lang_attr = 'en' if language == None else (language | replace('_', '-')) %}
+{%- set sphinx_writer = 'writer-html5' if html5_doctype else 'writer-html4' %}
```

Comment:
An alternative would be to use a `sphinx2` class. If we ever have similar styling incompatibilities with e.g. Sphinx 2.6 or 3.0, they will still use the HTML5 writer and we will need a different class to differentiate.

Reply:
True, I could see the writer output changing in the future and we might need more granularity eventually. In this case, you _can_ still use the html 4 writer with Sphinx 2, so the most truthy check would be the writer class (`html5_doctype` seemed to be the closest option already in the context data passed by sphinx).

When we cross it, I could see us needing logic like that also uses the sphinx version (assuming there is some major backwards incompatible change at some point)